frontend/src/metabase/visualizations/components/settings/ChartNestedSettingSeries.styled.tsx

```diff
@@ -14,4 +15,18 @@ export const OptionsIcon = styled(Icon)`
 
 export const SeriesNameInput = styled(InputBlurChange)`
   width: auto;
+
+  ${({ subtitle }) =>
+    subtitle &&
+    `
+    ${Input.Field} {
+      padding-top: 1rem;
+      padding-bottom: 0.375rem;
+    }
+
+    ${Input.Subtitle} {
+      top: 0.375rem;
+      left: 0.875rem;
```

Comment:
`0.8rem` seems to make subtitle align with input value a bit better.

<img width="320" alt="image" src="https://user-images.githubusercontent.com/380816/208500907-7e7890c9-df14-4ddd-87b5-2ece84352d8d.png">

Reply:
I think it's going to depend a lot on the characters. I did a bit of testing, sometimes it was better, sometimes it wasn't. But lets set it to 0.8rem and see what people think!